post.php

```diff
@@ -7,17 +7,6 @@
 require_once 'inc/anti-bot.php';
 require_once 'inc/bans.php';
 
-// Fix for magic quotes
-if (get_magic_quotes_gpc()) {
```

Comment:
Do we know this is safe to delete?

Reply:
Yes. It [always returns false](https://www.php.net/manual/en/function.get-magic-quotes-gpc.php).

This code basically says
```php
if (false) { ... }
```